wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -890,8 +890,9 @@ public void onCreate() {
 
             DapiClient client = PlatformRepo.getInstance().getPlatform().getClient();
             client.setSimplifiedMasternodeListManager(application.getWallet().getContext().masternodeListManager, masternodes);
-            client.getDapiAddressListProvider().addBannedAddress("211.30.243.82");
-            client.getDapiAddressListProvider().addBannedAddress("54.184.89.215");
+            client.getDapiAddressListProvider().addBannedAddress("45.48.168.16");
+            client.getDapiAddressListProvider().addBannedAddress("71.239.154.151");
+            client.getDapiAddressListProvider().addBannedAddress("174.34.233.98");
```

Comment:
There are many masternodes on these IP's that do not have platform.

This is a temporary solution for testnet.